tests/components/controlBar-test.js

```diff
@@ -934,14 +952,92 @@ describe('ControlBar', function () {
     expect(ccButtons.length).toBe(1);
   });
 
+  it('shows/hides multiaudio button if multiaudio available', function() {
+    var mockController = {
```

Comment:
Please use `baseMockController` and `baseMockProps` instead of defining a new object on each test.

Reply:
\+